.github/workflows/continuous-integration.yml

```diff
@@ -88,17 +98,10 @@ jobs:
         cmake --build . --config Release -- /maxcpucount:2
         
     - name: Install opensim-core
```

Comment:
It's not great that there is a separate Install step for one platform but not the others.

Reply:
Removed. Thanks @chrisdembia